minikube/vault/auth.tf

```diff
@@ -9,4 +9,21 @@ resource "vault_kubernetes_auth_backend_config" "example" {
   token_reviewer_jwt = var.kubernetes_reviewer_jwt
   issuer = "kubernetes.io/serviceaccount"
   disable_iss_validation = true
-}
\ No newline at end of file
+}
+
+resource "vault_auth_backend" "userpass" {
+  type = "userpass"
+}
+
+resource "vault_generic_endpoint" "monetr-user" {
+  depends_on           = [vault_auth_backend.userpass]
+  path                 = "auth/userpass/users/monetr"
+  ignore_absent_fields = true
+
+  data_json = jsonencode({
+    policies = [
+      vault_policy.rest-api-service-policy.name,
+    ]
+    password = "password"
+  })
+}
```

Comment:
Add a comment here explaining what this is for. It's just for local development. But make that clear.